docs/gettingstarted.md

```diff
@@ -0,0 +1,58 @@
+
+# Pre-req
+
+* Linux host with Docker 19.x or newer with Docker Swarm enabled
+    [Getting Started](https://docs.docker.com/get-started/)
+* A Splunk index for metrics typically "em_metrics"
```

Comment:
Please clarify a bit. Does this mean the Admin has to create an "em_metrics" index or can they expect that this is already there for other reasons?

Reply:
"em_metrics" is quasi standard the IAI and ITSI apps use it

Reply:
Admin must create it.  Unlike "main" (which is created automatically for event data), there is no automatically-created, default index for metrics.